src/api/transaction.rs

```diff
@@ -101,25 +100,22 @@ struct EncryptedAmountTransferredReceiverMetadata {
 
 #[derive(SerdeSerialize)]
 struct TransferredToEncryptedMetadata {
-    amount:               Amount,
     new_encrypted_amount: EncryptedAmount<EncryptedAmountsCurve>,
 }
 
 #[derive(SerdeSerialize)]
 struct TransferredToPublicMetadata {
     address:              AccountAddress,
-    amount:               Amount,
     new_encrypted_amount: EncryptedAmount<EncryptedAmountsCurve>,
     encrypted_amount:     EncryptedAmount<EncryptedAmountsCurve>,
     up_to_index:          EncryptedAmountAggIndex,
 }
 
 #[derive(SerdeSerialize)]
 struct TransferredWithScheduleMetadata {
-    receiver_address: AccountAddress,
-    amounts:          Vec<(Timestamp, Amount)>, // TODO convert to map?
+    amounts: Vec<(Timestamp, Amount)>, // TODO convert to map?
```

Comment:
I don't think this todo should be done. The list of amounts will always be ordered  by increasing timestamp so search is already easy with binary search.

Reply:
The problem I have with this is that it serializes to JSON like

```
"amounts": [
  [
    1665403200000,
    "100000000"
  ],
  ...
]
```

which does not have a clear meaning. I don't think it the map suggestion in the TODO is much clearer after all so will suggest that we instead change it to

```
"amounts": [
  {
    unix_time: 1665403200, // ok to use number
    amount: "100000000"
  },
  ...
]
```

Reply:
Why is it OK to use the large number for `unix_time_ms`? Javascript will have problems parsing this.

Reply:
Otherwise I like the suggestion.

Reply:
JS parses numbers correctly up to around 16 digits, so this is safe for the "foreseeable" future :) Rosetta already uses number for timestamp in the reponse of [`/network/status`](https://www.rosetta-api.org/docs/models/NetworkStatusResponse.html) so at least it wouldn't introduce new inaccuracies.